website/events/forms.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, *args, **kwargs):
             self.fields[key].initial = field["value"]
 
     def field_values(self):
-        for key, field in self.information_fields.items():
+        for key, __ in self.information_fields.items():
```

Comment:
Looping through keys is what happens when you do `for key in dict`, so the .items() should be removed here